### PR TITLE
global: Black support

### PR DESCRIPTION
**Pull request by diegodelemos**
Addresses reanahub/reana#329

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,21 @@
 Changes
 =======
 
+Version master (unreleased)
```
Comment (mvidalgarcia) [inline CHANGES.rst:4]: We should decide on a common header. What about `Unreleased`? Capital U.

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+---------------------------
+
+- Adds new utility to send emails.
+- Adds centralised operational options validation.
+- Fixes `memory link in Bravado client instantiation <https://github.com/reanahub/reana-server/issues/225>`_.
+- Makes maximum number of running workflows configurable.
+- Adds configurable prefix for component names.
+- Allow specifying unpacked Docker images.
```
Comment (mvidalgarcia) [inline CHANGES.rst:12]:
```suggestion
- Allows specifying unpacked Docker images.
```

File: CHANGES.rst [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+- Upgrades minimum version of Kubernetes Python library to 11.
+- Centralises CephFS PVC name.
+- Updates to latest CVMFS CSI driver.
+- Increases default log level to `INFO`.
```
Comment (mvidalgarcia) [inline CHANGES.rst:16]:
```suggestion
- Increases default log level to ``INFO``.
```

File: CHANGES.rst [hunk 4]
```diff
@@ -1,6 +1,21 @@
 Changes
 =======
 
+Version master (unreleased)
+---------------------------
+
+- Adds new utility to send emails.
+- Adds centralised operational options validation.
+- Fixes `memory link in Bravado client instantiation <https://github.com/reanahub/reana-server/issues/225>`_.
```
Comment (tiborsimko) [inline CHANGES.rst:9]: If we are going to link to GitHub issues in release notes, then let's do it uniformly everywhere, e.g. the following style:

```
- Adds new utility to send emails. (#124)
- Fixes memory link in Bravado client instantiation. (#225)
- ...
```

It's a style used in several projects including K8s.

File: CHANGES.rst (hunk 1 above)
Comment (tiborsimko) [reply CHANGES.rst:4]: In accordance with `FIXME` and `CHANGEME`, I'd go with `UNRELEASED` all caps.

File: docs/conf.py [hunk 5]
```diff
@@ -34,35 +34,35 @@
...
-master_doc = 'index'
+master_doc = "index"
 
 # General information about the project.
-project = 'reana'
-copyright = '2018, info@reana.io'
-author = 'info@reana.io'
+project = "reana"
+copyright = "2018, info@reana.io"
+author = "info@reana.io"
```
Comment (tiborsimko) [inline docs/conf.py:65]: BTW we could switch to `team@reana.io` everywhere, but we can also do it later, e.g. when we'll be globally updating copyright years one day.

File: docs/conf.py [hunk 6]
```diff
@@ -197,12 +190,18 @@
...
-    (master_doc, 'reana', 'reana Documentation',
-     author, 'reana', 'One line description of project.',
-     'Miscellaneous'),
+    (
+        master_doc,
+        "reana",
+        "reana Documentation",
+        author,
+        "reana",
+        "One line description of project.",
```
Comment (tiborsimko) [inline docs/conf.py:199]: Another thing for later: expand "One line description of project" :)

**Pull request merged**